hvac/api/auth_methods/aws.py

```diff
@@ -146,34 +151,44 @@ def configure_identity_integration(
             set up for the role name will still be valid
         :type iam_alias: str | unicode
+        :param iam_metadata: The metadata to include on the token returned by the login endpoint.
+            This metadata will be added to both audit logs, and on the ``iam_alias``. By default, it includes ``account_id``
+            and ``auth_type``. Additionally, ``canonical_arn``, ``client_arn``, ``client_user_id``, ``inferred_aws_region``, ``inferred_entity_id``,
+            and ``inferred_entity_type`` are available. To include no metadata, set to an empty list ``[]``.
+            To use only particular fields, select the explicit fields. To restore to defaults, send only a field of ``default``.
+            Only select fields that will have a low rate of change for your ``iam_alias`` because each change triggers a storage
+            write and can have a performance impact at scale.
+        :type iam_metadata: str | unicode, list
         :param ec2_alias: Configures how to generate the identity alias when using the ec2 auth method. Valid choices
             are role_id, instance_id, and image_id. When role_id is selected, the randomly generated ID of the role is
             used. When instance_id is selected, the instance identifier is used as the identity alias name. When
             image_id is selected, AMI ID of the instance is used as the identity alias name
         :type ec2_alias: str | unicode
+        :param ec2_metadata: The metadata to include on the token returned by the login endpoint. This metadata will be
+            added to both audit logs, and on the ``ec2_alias``. By default, it includes ``account_id`` and ``auth_type``. Additionally,
+            ``ami_id``, ``instance_id``, and ``region`` are available. To include no metadata, set to an empty list ``[]``.
+            To use only particular fields, select the explicit fields. To restore to defaults, send only a field of ``default``.
+            Only select fields that will have a low rate of change for your ``ec2_alias`` because each change triggers a storage
+            write and can have a performance impact at scale.
+        :type ec2_metadata: str | unicode, list
         :param mount_point: The path the AWS auth method was mounted on.
         :type mount_point: str | unicode
         :return: The response of the request
         :rtype: request.Response
         """
         if iam_alias is not None and iam_alias not in ALLOWED_IAM_ALIAS_TYPES:
-            error_msg = 'invalid iam alias type provided: "{arg}"; supported iam alias types: "{alias_types}"'
-            raise exceptions.ParamValidationError(
-                error_msg.format(
-                    arg=iam_alias, environments=",".join(ALLOWED_IAM_ALIAS_TYPES)
-                )
-            )
+            error_msg = f'invalid iam alias type provided: "{iam_alias}"; supported iam alias types: "{",".join(ALLOWED_IAM_ALIAS_TYPES)}"'
```

Comment:
f-strings only work with double quotes. You can make the quotes inside the strings into single quotes `'` or you can use an alternative method to get double-quotes in there: https://stackoverflow.com/q/17085354/3905079

As is, it's being treated by multiple statements separated by semi-colon, which is why flake8 is failing.

Reply:
Thanks for catching that, not sure why flake8 didn't catch it when I ran it on my end.